index.html

```diff
@@ -3,14 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
-    <title>My first Microverse student project</title>
-
+    <title>Luis's portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link rel="stylesheet" href="css/style.css">
   </head>
-  <body>
 
-    <h1>Hello Microverse!</h1>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <a href="#home" class="nav-logo">My Logo</a>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact" class="nav-link">Contact</a>
+          </li>
+        </ul>
+        <div class="mobile-menu"></div>
+      </nav>
+    </header>
 
+    <section class="main">
+      <h1 class="intro">
+        I'm Luis<br>Glad to see you!
+      </h1>
+      <h2 class="description">
+        I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </h2>
```

Comment:
- [ ] **REQUIRED:** The paragraph should be inside the `p` tag, your way is not true for semantic HTML, so let's change it to the `p` tag.